debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -704,6 +704,22 @@ public JdbcConnection prepareQueryWithBlockingConsumer(String preparedQueryStrin
         return this;
     }
 
+    /**
+     * Execute a SQL prepared query.
+     *
+     * @param preparedQueryString the prepared query string
+     * @param preparer the function that supplied arguments to the prepared statement; may not be null
+     * @param resultConsumer the consumer of the query results
+     * @return this object for chaining methods together
+     * @throws SQLException if there is an error connecting to the database or executing the statements
+     * @see #execute(Operations)
+     */
+    public JdbcConnection prepareQuery(String preparedQueryString) throws SQLException {
```

Comment:
I think JavaDoc is unrelated to the method.

Reply:
Mh, yeah, copy and paste. Will fix.